src/main/java/net/lenni0451/spm/commands/subs/Download_Sub.java

```diff
@@ -18,13 +18,19 @@ public class Download_Sub implements ISubCommandMultithreaded {
 
     @Override
     public boolean execute(CommandSender sender, String[] args) {
-        if (args.length != 3) return false;
+        if (args.length < 3) return false;
         if (!args[0].equalsIgnoreCase("direct") && !args[0].equalsIgnoreCase("spigot")) return false;
         if (!sender.hasPermission("pluginmanager.commands.download." + args[0].toLowerCase())) {
             Logger.sendPermissionMessage(sender);
             return true;
         }
 
+        String filename = args[2];
+        if(args.length > 3 && filename.startsWith("\"") && args[args.length-1].endsWith("\"")) {
+        	for(int i = 3; i < args.length; i++) {
+        		filename += args[i];
+        	}
```

Comment:
You need to remove the leading and trailing " here

Reply:
Done on line 34